documentation/pom.xml

```diff
@@ -42,6 +42,7 @@
 
         <forbiddenapis-junit.path>forbidden-allow-junit.txt</forbiddenapis-junit.path>
         <hibernate-validator-parent.path>..</hibernate-validator-parent.path>
+        <automatic.module.name>org.hibernate.validator.referenceguide</automatic.module.name>
```

Comment:
I don't think this one is necessary: we don't publish anything and it's not supposed to be used.